functions.php

```diff
@@ -19,24 +19,20 @@ function esc($text) {
 date_default_timezone_set("Europe/Moscow");
 
 function get_time_end($time) {
-
     $time_interval = strtotime($time) - time();
     $hours_end = floor($time_interval / 3600);
     $mins_end = floor(($time_interval % 3600) / 60);
     $time_end = [$hours_end, $mins_end];
```

Comment:
`floor` возвращает тип float, а мы ожидаем целые числа, поэтому предлагаю привести тип явно
```suggestion
        $time_end = [ intval($hours_end), intval($mins_end) ];
```